core/java/iesi-rest-without-microservices/src/main/java/io/metadew/iesi/server/rest/script/ScriptController.java

```diff
@@ -70,19 +91,18 @@ public ScriptDto post(@Valid @RequestBody ScriptPostDto scriptPostDto) throws Me
         for (ScriptPostDto scriptPostDto : scriptDtos) {
             halMultipleEmbeddedResource.embedResource(scriptPostDto);
         }
-
         return halMultipleEmbeddedResource;
     }
 
     @PutMapping("/{name}/{version}")
-    public ScriptDto put(@PathVariable String name, @PathVariable Long version,
+    public ScriptDto put(@PathVariable String name,
+                         @PathVariable Long version,
                          @RequestBody ScriptPostDto scriptPostDto) throws MetadataDoesNotExistException {
         if (!scriptPostDto.getName().equals(name)) {
             throw new DataBadRequestException(name);
         }
         scriptService.updateScript(scriptPostDto);
         return scriptDtoModelAssembler.toModel(scriptPostDtoService.convertToEntity(scriptPostDto));
-
     }
```

Comment:
The version parameter is never used here.
I think it should be used to know if a user wants to update an existing version of a script or to create a new version.

Reply:
Added a check on version